docs/commands/GetArchiveEvents.md

```diff
@@ -0,0 +1,105 @@
+# GetArchiveEvents
+
+Command to request/receive events from device archive.
+
+
+## Request
+
+### Format
+
+| Size   | Type        | Field                                  |
+| ------ | ----------- | -------------------------------------- |
+| `1`    | `byte`      | command id = `0x0b`                    |
+| `1`    | `byte`      | command size = `5`                     |
+| `4`    | `uint32_be` | [start time](../types.md#time-2000) |
+| `1`    | `byte`      | [events](#events)                      |
+
+It's a command with a [two-bytes header](../message.md#command-with-a-two-bytes-header).
+
+### Parameters
+
+#### **start time**
+
+Start time in seconds to read from archive. In [time 2000](../types.md#time-2000) format.
+
+#### **events**
+
+Amount of events to get from archive.
+
+### Examples
+
+#### channels `1`, `3`, `4`:
+
+| Field        | Value                     | Bits                                 | Hex          |
+| ------------ | ------------------------- | ------------------------------------ | -----------  |
+| command id   | `11`                      |                                      | `0x0b`       |
+| command size | `5`                       |                                      | `0x05`       |
+| start time   | `2023.04.03 14:01:17 GMT` | `0b00101011101111011001100010101101` | `0x2bbd98ad` |
```

Comment:
do we need bits here? let's do it as in Time2000.md

maybe we can put both `2023.04.03 14:01:17 GMT` and seconds in Value column

Reply:
bits column - deleted, seconds with date representation added